### DISCUSSION TITLE
is styled-system a dead project?

### DISCUSSION THREAD

**Discussion by bstro** [chakra-ui/chakra-ui#4614] [Q&A] [answered]
Hi there, while evaluating `chakra-ui` for use on my team, I've noticed that one of chakra's core dependencies, styled-system, seems like it might possibly be a dead project.

[link 1](https://github.com/styled-system/styled-system/issues/1986)
[link 2](https://github.com/styled-system/styled-system/issues/1547)

How coupled to styled-system is chakra, and has there been any discussion about finding an alternative for future versions? 

Thanks!

**Comment by mdovn** [comment]
Yeah, looks like it was abandoned. We should fork or migrate (carefully choose) to another one

**Comment by tony** [comment]
Someone made a pure TypeScript styled-system fork: https://github.com/spicy-ui/styled-system

They use it in their pure typescript react framework: https://github.com/spicy-ui/core

**Comment by daniel-schulz-codes** [comment, marked as answer]
Hej @bstro 

AFAIK we don't use `styled-system/styled-system` in chakra - we use `@chakra/styled-system` which is fully maintained by us.

**Reply by bstro** [reply]
Thanks! This was super helpful, I missed that detail.

**Reply by Brennvo** [reply]
Is it unheard of to use `@chakra/styled-system` as an aid in developing a design system that can't use the main Chakra UI components? I really enjoy the composability of the styles in this library, and given that `styled-system/styled-system` isn't maintained, the chakra version sounds like a really great option.
